package.json

```diff
@@ -76,7 +76,7 @@
   },
   "repository": "forcedotcom/sfdx-scanner",
   "scripts": {
-    "build": "./gradlew build",
+    "build": "./gradlew build test",
```

Comment:
?  We should not intermingle building with running tests here.  In the CI for example, we run build, then we run test, then we run the linter.

Instead, consider adding it to the test command below:

 "test": "./gradlew test && nyc mocha './test/**/*.test.ts'"

Reply:
Good call. Will fix this.